keps/754-time-dependent-job-priority/README.md

```diff
@@ -0,0 +1,197 @@
+# KEP-754: Time-dependent job priority
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [Story 1](#story-1)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Kueue WorkloadPriorityClass API](#kueue-workloadpriorityclass-api)
+  - [How to use WorkloadPriorityClass on Job](#how-to-use-workloadpriorityclass-on-job)
+  - [Priority Calculation](#priority-calculation)
+  - [Other points](#other-points)
+  - [Test Plan](#test-plan)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+This proposal allows job priorities to be dynamically adjusted based on time.  
+This functionality aims to prioritize the execution of jobs that have been waiting 
+for an extended period, even if they have a lower initial priority.  
+By introducing time-dependent priority adjustments, the system can ensure that jobs 
```

Comment:
What about introducing just a priority/ordering annotation and keeping the updates outside of Kueue, in a simple controller? I can imagine quite a few different use cases for custom ordering in a queue and obviously not all of them can be added to the API and implemented.